netci.groovy

```diff
@@ -221,34 +231,45 @@ def CreateStyleCheckTasks = { taskString, taskName, checkName ->
 // INNER LOOP TASKS
 // ----------------
 
-CreateBuildTasks('Windows_NT', null, null, "-winBlue", true, null, null)
+CreateBuildTasks(latestWindowsMachine, latestWindowsMachineTag, null, null, "-winBlue", true, null, null)
 
 // Add some additional daily configs to trigger per-PR as a quality gate:
 // x64_debug Slow Tests
 CreateBuildTask(true, 'x64', 'debug',
-    'Windows_NT', 'ci_slow', null, '-winBlue -includeSlow', false, null, null)
+    latestWindowsMachine, latestWindowsMachineTag, 'ci_slow', null, '-winBlue -includeSlow', false, null, null)
 // x64_debug DisableJIT
 CreateBuildTask(true, 'x64', 'debug',
-    'Windows_NT', 'ci_disablejit', '"/p:BuildJIT=false"', '-winBlue -disablejit', false, null, null)
+    latestWindowsMachine, latestWindowsMachineTag, 'ci_disablejit', '"/p:BuildJIT=false"', '-winBlue -disablejit', false, null, null)
 // x64_debug Lite
 CreateBuildTask(true, 'x64', 'debug',
-    'Windows_NT', 'ci_lite', '"/p:BuildLite=true"', '-winBlue -lite', false, null, null)
+    latestWindowsMachine, latestWindowsMachineTag, 'ci_lite', '"/p:BuildLite=true"', '-winBlue -lite', false, null, null)
+// x64_debug Legacy
+CreateBuildTask(true, 'x64', 'debug',
```

Comment:
So ci_lite and ci_disablejit don't get rolling builds. Is that intentional?

Reply:
There's no need for rolling builds since they're intended to be daily builds. The PR-only tasks for these daily build configs were added as a minimal sanity check. This is by design. We can revisit this (or the general distinction of daily|rolling) as a separate discussion and work item if you think that's worthwhile.